packages/turf-helpers/index.ts

```diff
@@ -665,6 +665,18 @@ export function bearingToAzimuth(bearing: number): number {
   return angle;
 }
 
+/**
+ * Converts any azimuth angle from the north line direction (positive clockwise)
+ * and returns an angle between -180 and +180 degrees (positive clockwise), 0 being the north line
+ *
+ * @name azimuthToBearing
+ * @param {number} angle between 0 and 360 degrees
+ * @returns {number} bearing between -180 and +180 degrees
+ */
+export function azimuthToBearing(angle: number): number {
+  return angle < 0 ? angle + 360 : angle;
```

Comment:
Pretty sure this isn't working as expected @basvdijk now that I've run CI and seen the failing tests.

Reply:
this works 😃:
```js
function azimuthToBearing(angle) {
  angle = angle % 360;
  if (angle > 0) return angle > 180 ? angle - 360 : angle;
  return angle < -180 ? angle + 360 : angle;
}

test('azimuthToBearing', t => {
  t.equal(azimuthToBearing(0), 0);
  t.equal(azimuthToBearing(360), 0);
  t.equal(azimuthToBearing(180), 180);

  t.equal(azimuthToBearing(40), 40);
  t.equal(azimuthToBearing(40 + 360), 40);

  t.equal(azimuthToBearing(-35), -35);
  t.equal(azimuthToBearing(-35 - 360), -35);

  t.equal(azimuthToBearing(255), -105);
  t.equal(azimuthToBearing(255 + 360), -105);

  t.equal(azimuthToBearing(-200), 160);
  t.equal(azimuthToBearing(-200 - 360), 160);

  t.end();
});
```